tests/integration/targets/inventory/.sops.yaml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
The directory must be called `inventory_sops`, since the plugin's name is `sops`.

Reply:
ok